kotlin/README.md

```
@@ -328,10 +328,11 @@ val issuerMetadata = IssuerMetaData(
                         CREDENTIAL_ENDPOINT, 
                         DOWNLOAD_TIMEOUT, 
                         DOC_TYPE,
-                        CLAIMS, 
                         CredentialFormat.MSO_MDOC )
 ```
 
+>  **Note**: The `claims` parameter is no longer included in the credential request body for `mso_mdoc` format as per the updated OID4VCI specification. Even if `claims` is provided in `IssuerMetaData`, it will be ignored when constructing the credential request for `mso_mdoc` format.
```

markdownlint-cli2 (0.18.1)

Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)